app/src/main/res/drawable/nps_survey_button_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners android:radius="@dimen/spotlight_done_button_corner_radius" />
```

Comment:
Prefer creating a dedicated dimension here, instead of using one named after a different screen, per #2399.

Reply:
Done.